website/common/script/cron.js

```diff
@@ -6,6 +6,7 @@
  */
 import _ from 'lodash';
 import moment from 'moment';
+import 'moment-recur';
```

Comment:
also it doesn't seem very updated https://github.com/c-trimm/moment-recur/issues